### PR TITLE
Remove reference to "responsible document"

### LINKED ISSUE #61: Broken references in Mixed Content
While crawling [Mixed Content](https://w3c.github.io/webappsec-mixed-content/), the following links to other specifications were detected as pointing to non-existing anchors, which should be fixed:
* [ ] https://html.spec.whatwg.org/multipage/webappapis.html#responsible-document

<sub>This issue was detected and reported semi-automatically by [Strudy](https://github.com/w3c/strudy/) based on data collected in [webref](https://github.com/w3c/webref/).</sub>

### PR DESCRIPTION
**Pull request by carlosjoan91**
Fixes #61

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -324,18 +324,15 @@ type: dfn
         [=potentially trustworthy origin=], then return
         "`Prohibits Mixed Security Contexts`".
 
-    2.  If |settings| has a <a>responsible document</a> |document|, then:
+    2.  If |settings|' [=environment settings object/global object=] has a
```
Comment (annevk) [inline index.bs:327]:
```suggestion
    2.  If |settings|' [=environment settings object/global object=] has an
```

**Review by annevk — APPROVED** [review]
This looks good. Might be wise to let @domenic double check though.

**Comment by carlosjoan91** [comment]
Thanks for reviewing! @domenic could you take a look before I merge this? Thanks!

File: index.bs [hunk 2]
```diff
@@ -324,18 +324,15 @@ type: dfn
         [=potentially trustworthy origin=], then return
         "`Prohibits Mixed Security Contexts`".
 
-    2.  If |settings| has a <a>responsible document</a> |document|, then:
+    2.  If |settings|' [=environment settings object/global object=] has an
+        [=Window/associated Document=] |document|, then:
```
Comment (domenic) [inline index.bs:328]: It'd be slightly more correct to ask if settings's global object is a Window, and only then look at its associated Document. But what you have here seems reasonable too.

**Review by domenic — APPROVED** [review]

File: index.bs (hunk 2 above)
Comment (carlosjoan91) [reply index.bs]: Done, thanks for reviewing.

**Pull request merged**